modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc

```diff
@@ -30,3 +30,9 @@ that is not supported. The following Tuned plug-ins are currently not supported:
 * bootloader
 * script
 * systemd
+
+See
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8-beta/html/monitoring_and_managing_system_status_and_performance/customizing-tuned-profiles_monitoring-and-managing-system-status-and-performance#available-tuned-plug-ins_customizing-tuned-profiles[Available
```

Comment:
Do we already know what will be the links for non-beta RHEL8?  Will 8-beta links go away once it is released?  Thanks!

Reply:
@jmencak I confirmed with the RHEL docs team that the /8-beta/ part of the file path will become /8/ upon release. They have some new docs tooling, so I don't want to assume that redirects for the beta to GA docs will be put into place. We may have to update these paths manually. 

@vikram-redhat I had some discussion with Steven from RHEL docs. Do you happen to know anything about redirects for RHEL 8 beta docs?

Reply:
@ahardin-rh - no idea. I assume they will put up redirects. I can ask Tomas Capek and update this thread.

Reply:
@vikram-redhat Thank you!

Reply:
@ahardin-rh Confirmed with Jana Wirch (RHEL CS) that there will be **no** redirects for RHEL 8 beta docs to RHEL 8 docs.

Can you confirm with Steven what the new link for this doc might be? They GA before OpenShift GA.

Reply:
@vikram-redhat Thanks for the update! I confirmed /8-beta/ will become /8/ upon release, so I updated the file path here. 

@mffiedler Can you please provide QE review? Thanks!